extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -291,6 +296,23 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
     /** @private {number} */
     this.ifi_ = 0;
+
+    /** @private {boolean} */
+    this.isFluid_ = false;
+
+    /** @private {?string} */
+    this.fluidImpressionUrl_ = null;
+
+    /**
+     * Used to style fluid ad slot, if no width is provided.
+     * @private {number}
+     */
+    this.parentWidth_ = 0;
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return isLayoutSizeDefined(layout) || layout == Layout.FLUID;
```

Comment:
Might as well invert the order here saving execution of isLayoutSizeDefined (I'm sure it has no cost but still...)

Reply:
Done.